{{cookiecutter.project_name}}/default.nix

```diff
@@ -26,6 +26,7 @@ let
       pkgs.haskellPackages.ghcid
       pkgs.haskellPackages.ormolu
       pkgs.haskellPackages.hlint
+      {% if cookiecutter.use_hpack == "y" %}pkgs.haskellPackages.hpack{% endif %}
```

Comment:
The nix build will pick this up automatically, but it would be nice to have in the shell for local dev

Reply:
I agree, it is definitely good to have. But even after this one needs to remember running `hpack` every time after `package.yaml`. So it would be great if we can also mention this on the README.

Reply:
Handled in https://github.com/utdemir/hs-nix-template/pull/21/commits/1aeab4d448f93f440e30bc1514a918c2993edf8c